openedx/core/djangoapps/theming/template_loaders.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, engine, dirs=None):
             self.dirs = theme_dirs + self.dirs
         super().__init__(engine, self.dirs)
 
+    def get_dirs(self):
+        """
+        Override get_dirs method.
+
+        Make the theme templates a priority, avoiding cashing templates for django ones.
+        """
+        self.dirs = self.engine.dirs
+        theme_dirs = self.get_theme_template_sources()
+        if isinstance(theme_dirs, list):
```

Comment:
nit: It looks like all the code paths for `get_theme_template_sources` will return a False or list of dirs.. so this can be simply:
```suggestion
        if theme_dirs:
```